hasura/migrations/1622090276691_require_player_ethaddress/down.sql

```diff
@@ -0,0 +1,18 @@
+
+alter table "public"."player_skill" drop constraint "player_skill_player_id_fkey",
+          add constraint "Player_Skill_player_id_fkey"
+          foreign key ("player_id")
+          references "public"."player"
+          ("id")
+          on update no action
+          on delete no action;
+
+alter table "public"."player_account" drop constraint "player_account_player_id_fkey",
+          add constraint "Account_player_id_fkey"
+          foreign key ("player_id")
+          references "public"."player"
+          ("id")
+          on update no action
+          on delete no action;
```

Comment:
Basically deletes a players related accounts and skills when a player gets deleted. Makes it simpler to do delete operations